conanfile.py

```diff
@@ -27,6 +27,7 @@ class Clio(ConanFile):
         'grpc/1.50.1',
         'openssl/1.1.1u',
         'xrpl/2.0.0-rc1',
+        'libbacktrace/cci.20210118'
```

Comment:
Is this needed with Boost.Stacktrace still?

Reply:
As I understand Boost.Stacktrace is a wrapper around this library (if it is used in  backtrace mode).